# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/ion/bug1811803.js
# Commit: a1ca52a54ce8
# Full Hash: a1ca52a54ce80493e975350f1cc00f4eca15ed76
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2023-01-27 09:46:52
# Description:
#   Bug 1811803 - Use useBoxFixedAtStart in LMegemorphicSetElement r=jandem
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D167870
# ==============================================================================

diff -r b27d83c0aa57 -r a1ca52a54ce8 js/src/jit-test/tests/ion/bug1811803.js
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/js/src/jit-test/tests/ion/bug1811803.js	Thu Jan 26 18:46:08 2023 +0000
@@ -0,0 +1,5 @@
+// |jit-test| --fuzzing-safe; --ion-offthread-compile=off; --fast-warmup
+
+a = {}
+for (b = 0; b < 100; ++b)
+  a['x' + b] = 'x' + b;